docs/technical_reference/property_models/seawater_ion_generic.rst

```diff
@@ -0,0 +1,87 @@
+Seawater Ion Generic Property Package
+=====================================
+
+This package implements property relationships for artificial seawater as provided in `Islam et al. (2021) <https://www.sciencedirect.com/science/article/abs/pii/S1383586621009412>`_.
+
+This artificial seawater property package:
+   * supports only 'H2O', 'Na_+', 'Ca_2+', 'Mg_2+', 'Cl\_-', 'SO4_2-'
+   * supports only liquid phase
+   * is formulated on a molar basis
+   * does not support dynamics
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Components", ":math:`j`", "['H2O', 'Na_+', 'Ca_2+', 'Mg_2+', 'Cl\_-', 'SO4_2-']"
+   "Phases", ":math:`p`", "['Liq']"
+
+State variables
+---------------
+.. csv-table::
+   :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+   "Component molar flowrate", ":math:`N_j`", "flow_mol_phase_comp", "[p, j]", ":math:`\text{mole/s}`"
+   "Temperature", ":math:`T`", "temperature", "None", ":math:`\text{K}`"
+   "Pressure", ":math:`P`", "pressure", "None", ":math:`\text{Pa}`"
+
+Properties
```

Comment:
There are probably a bunch of parameters that we want list out here.

Reply:
@adam-a-a Parameters as in more state variables (I think these are the only 3?) or properties? Looking at the property package file, there are a few things I missed, such as vol_mol_pure, vol_mol_liq_comp, and relative_permittivity_liq_comp. There's no metadata list like with the other prop packs, so I wasn't sure exactly what to include here. Should I still include properties like mass flow rate and mass frac even though this prop pack is formulated on a molar basis?

Reply:
@MarcusHolly this one is a bit trickier because it is really a configuration file for the GenerericParameterBlock. However, we might want to think about this a little more since this was written a while back and would need to revisit in more detail. One thing I do know is that we'd want to point to the eNRTL parameters (`Liq_tau`) in some fashion, which was the main purpose of the effort.

Reply:
Added a few more properties, but we can revisit this later (after things have died down) to finish fleshing out this documentation.